### ISSUE TITLE
Can't move Unity Editor's windows

### ISSUE THREAD

**Issue by afk-mario** [swaywm/sway#5567]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on irc.freenode.net.
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Problems with the Wayland version of Firefox are likely to be Firefox bugs. Start by submitting your issue to the Firefox Bugzilla and come back here only after they confirm otherwise.

### Please fill out the following:
- **Sway Version:**

sway version 1.4-9cda5a5b (May  6 2020, branch 'master')

- **Debug Log:**

https://gist.github.com/afk-mario/e7245b870751305cb8b1614787081b98

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **Description:**
  - The steps you took in plain English to reproduce the problem.
Using Unity 2019.4.4f1 open the editor and try to move the editor tabs so that you can re-arrange them, nothing happens.
This seems to work fine using i3
![unity](https://user-images.githubusercontent.com/1025751/88117667-570e5c00-cb81-11ea-8a9e-7a24ca6a8e3d.gif)

**Comment by Geo25rey** [comment]
Does switching to floating mode change anything? The default keybind is $mod+Shift+Space.

**Comment by afk-mario** [comment]
@Geo25rey no, already tried different window modes

**Comment by lhanson** [comment]
I can also confirm that I see the same behavior on Sway 1.5 and Unity 2019.4.8f1.

**Comment by HunterretnuH** [comment]
I'm also experiencing the same problem.
Sway: 1.5.1
Unity: 2019.4.9f1

**Comment by tinkerttoy** [comment]
Fourthing, and I should add that having to switch to GNOME to get any work done is an indignity.
Sway: 1.5
Unity: 2019.4.10f1

**Comment by ewnu** [comment]
Further confirm that 1.6 version doesn't solve this issue either.

In addition, new windows opened inside unity editor either floating or tiled,
did not merge into main editor window but showed weird blank window while trying to move.

**Comment by FlyingHelix** [comment]
it is still an issue with sway 1.6.1 and unity 2020.3.26f1. Any updates on fixing this?

**Comment by emersion** [comment]
Patches are welcome.

**Comment by senexger** [comment]
Still an issue

Unity 2020.3.33f1
Sway 1.7

**Comment by acheronfail** [comment]
I've had some (hacky) success by temporarily adding 

```
for_window [instance="Unity"] floating enable
```

to my config, then re-arranging the panels (albeit with quite a few window positioning glitches), and then removing that from the sway config and `reload`ing.

Interesting that `i3` handles this perfectly without any config :thinking: I wonder what `sway`'s missing when these windows are clicked and dragged...

**Comment by SchnozzleCat** [comment]
Any updates on this? 

Even worse than the buggy Unity window layouting is that stuff like dropdowns are basically unusable without floating window mode enabled for the Unity instance.

Even with floating mode enabled, the dropdown always spawns in the center of the screen, making for a really strange workflow. 

This is even worse if the Unity windows itself doesn't cover the center of the screen, because then moving from the Unity window over to the dropdown will focus the window that is occupying the center, and as this is a window switch it will close the Unity dropdown again, making it impossible to click.

I'm not sure if i3 has the same issue with spawning the dropdowns in the wrong (i.e. always in the middle of the screen) location?

**Comment by melonmouse** [comment]
In case it helps anyone, here are all the (hacky) rules I've put in my `~/.config/sway/config` for Unity:

```
for_window [class="Unity" instance="Unity" title="Unity"] floating disable, resize set height 15 ppt
for_window [title="Unity.VisualScripting.FuzzyWindow"] floating enable
for_window [title="UnityEditor.AddComponent.AddComponentWindow"] floating enable
for_window [title="UnityEditor.AnnotationWindow"] floating enable
for_window [title="UnityEditor.PopupWindow"] floating enable, move position mouse
for_window [title="UnityEditor.Snap"] floating enable, move position mouse
for_window [title="UnityEditor.IMGUI.Controls.AdvancedDropdownWindow"] floating enable
for_window [title="^Color$"] floating enable
for_window [title="UnityEditor.Searcher.SearcherWindow"] floating enable
for_window [title="UnityEngine.InputSystem.Editor.AdvancedDropdownWindow"] floating enable
for_window [title="UnityEditor.Rendering.FilterWindow"] floating enable
for_window [title="Select Material"] floating enable
for_window [title="Gradient Editor"] floating enable
for_window [title="UnityEditor.LayerVisibilityWindow"] floating enable
```

Some of these are opinionated, so you may need to tweak them. These won't fix everything, but it helps.
